qrenderdoc/Windows/TextureViewer.h

```diff
@@ -133,6 +134,15 @@ class TextureViewer : public QFrame, public ITextureViewer, public ICaptureViewe
 
   Q_PROPERTY(QVariant persistData READ persistData WRITE setPersistData DESIGNABLE false SCRIPTABLE false)
 
+  // Texture List
+  enum class FilterType : uint_fast8_t
```

Comment:
This is a minor complaint but it's fine to pick a base type for the enum class here, but I'd rather have a more normal one like uint32_t since it's not an issue for storage/perf here :smile: . FYI it's optional to have a base type for enum classes, so you could omit it if you want too.